dv/uvm/core_ibex/scripts/compile-tb.py

```diff
@@ -35,6 +36,41 @@ def main() -> int:
 
     os.makedirs(args.output, exist_ok=True)
 
+    # Find the correct flags for the tb to link against the compiled ISS
+    cosim_subst_vars = {}
+    spike_iss_pc = ['riscv-riscv', 'riscv-disasm', 'riscv-fdt']
+    iss_pkgconfig_dict = {
+        'ISS_CFLAGS'  : ['--cflags'],
+        'ISS_LDFLAGS' : ['--libs-only-other'],
+        'ISS_LIBS'    : ['--libs-only-l', '--libs-only-L'],
+    }
+    if args.en_cosim:
+
```

Comment:
Nit: I think the conventional layout for these things wouldn't have the newline here.